app/src/main/res/layout/fragment_upload_media_detail_fragment.xml

```diff
@@ -80,6 +80,7 @@
                   android:clickable="false"
                   android:focusable="false"
                   android:padding="@dimen/medium_height"
+                  android:rotation="180"
```

Comment:
Inverting an inversion makes code more difficult to understand in the future.

How about inverting where each is called?

Or renaming the image files if that makds sense.

Reply:
The problem with renaming the image file is that it's being used in two other XML files, which would create confusion. Also, inverting where each is called is not a solution, as we need to invert the image only once, initially. Thus, I can add comments in the xml, explaining why we are rotating the image, or can rotate the image in onCreateView, but even there I'll have to add the same comments. @nicolas-raoul what should I do?

Reply:
What are the filenames involved?

Reply:
The image file (ic_expand_less_black_24dp) is used in 3 files:
1. fragment_upload_media_detail_fragment.xml (Fragment which has the current issue)
2. activity_upload.xml
3. nearby_filter_all_items.xml

Reply:
@nicolas-raoul What should be my next steps? I believe commenting about the rotation seems the best solution :)

Reply:
Sure, I'll change the name of the image file :]